src/pages/Landing/Hero.tsx

```diff
@@ -142,14 +148,20 @@ export const Hero = () => {
           <div className={`flex ${m ? 'flex-column is-mobile' : ''}`}>
             <ContentContainer className={m ? undefined : 'pt100'}>
               <LefContentContainer>
-                {isSmallScreen && (
+                {isMediumScreen && (
                   <TitleContainer>
                     <Animate enter="fadeIn" appear="fadeIn" delay={150}>
                       <LogoContainer className={m ? 'mb50' : undefined}>
-                        <EthLogo src={EthDiamondPlain} />
                         <LogoText>
-                          Eth2 Launch Pad{' '}
-                          {IS_MAINNET ? `` : `for ${ETH2_NETWORK_NAME} testnet`}
+                          {IS_MAINNET ? (
+                            <FormattedMessage defaultMessage="Eth2 Launchpad" />
+                          ) : (
+                            <FormattedMessage
+                              defaultMessage="Eth2 Launchpad for {TESTNET_LAUNCHPAD_NAME} testnet"
+                              values={{ TESTNET_LAUNCHPAD_NAME }}
```

Comment:
Again, this will only show up if user is on testnet, in which case `ETH2_NETWORK_NAME === TESTNET_LAUNCHPAD_NAME`... 
`TESTNET_LAUNCHPAD_NAME` is used here to emphasize that this will never say "mainnet"